src/main/java/com/firstspringmvcapp/dto/PersonDTO.java

```diff
@@ -0,0 +1,25 @@
+package com.firstspringmvcapp.dto;
+
+import jakarta.validation.constraints.*;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class PersonDTO {
+    @Min(value = 1, message = "Age should be greater than 0")
+    @Max(value = 110, message = "Aren't you a bit too old to use the internet?")
+    private int age;
```

Comment:
The common way is to use objects as Integer, but I think you could leave int if you want to